Automattic/woocommerce-payments
# Fix modal header alignment on safari

Fixes #5548 

#### Changes proposed in this Pull Request

Fix header alignment for the "Enable Manual Capture" modal on the Safari browser. 

Current CSS relies on complex margin and padding rules overrides that include negative margins. Safari doesn't recognize them well, so I've refactored the CSS to avoid negative margins. 

The overall design is exactly the same.

Before:
![image](https://github.com/Automattic/woocommerce-payments/assets/24650518/491d766b-29f7-4e10-923b-80501dceafb8)

After:
![image](https://github.com/Automattic/woocommerce-payments/assets/24650518/1260694c-59d6-4b46-94b9-6b4f79eb754f)

#### Testing instructions

* Open payments->settings in Safari browser
* Click "Issue an authorization on checkout, and capture later" checkbox
* Observe the modal header

-------------------

- [ x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ x] Covered with tests (or have a good reason not to test in description ☝️)
- [ x] Tested on mobile (or does not apply)

**Post merge**
